Pidaq/Lib/uploadToDrive/uploadToDrive.py

```diff
@@ -0,0 +1,125 @@
+from __future__ import print_function
+import os
+import logging
+from datetime import datetime
+import json

```

Comment:
I will fix that. Regarding the `parentDictionary`, I initially had it inside the function but decided to take it out because I am calling it from outside the file in RunInCmd.py. I edited the code to display the keys in the dictionary (to show the folders available) to whoever is uploading.